Apps/phcrowdstrikeoauth/crowdstrikeoauthapi.json

```diff
@@ -1513,6 +1600,5736 @@
+                    "column_name": "hash",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data.*.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.id",
+                    "data_type": "numeric",
+                    "example_values": [
+                        2
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.size",
+                    "data_type": "numeric",
+                    "example_values": [
+                        5122
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.file_hash",
+                    "data_type": "string",
+                    "example_values": [
+                        "53d84902e0a25be8706df19506f78799deab0082149b926b4117270f9c8673ad"
+                    ],
+                    "contains": [
+                        "sha256"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.file_name",
+                    "data_type": "string",
+                    "contains": [
+                        "filename"
+                    ],
+                    "example_values": [
+                        "test"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.session_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "b2403653-1294-488e-be81-7aadb69b52f1"
+                    ],
+                    "contains": [
+                        "crowdstrike rtr session id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.vault_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "30c5e524e975816fbce1d958150e394efc219772"
+                    ],
+                    "contains": [
+                        "sha1",
+                        "vault id"
+                    ]
+                }
+            ],
+            "versions": "EQ(*)",
+            "render": {
+                "title": "Get Session File",
+                "type": "table",
+                "height": 5,
+                "width": 12
+            }
+        },
+        {
+            "action": "set status",
+            "description": "Set the state of a detection in Crowdstrike Host",
+            "verbose": "The detection <b>id</b> can be obtained from the Crowdstrike UI.<br>Valid values for the <b>state</b> parameter are <i>new</i>, <i>in_progress</i>, <i>true_positive</i>, <i>false_positive</i> and <i>ignored</i>.",
```

Comment:
Is it necessary to list out the `state` values allowed if they are in a `value_list`?